phpBB/phpbb/storage/adapter/local.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\storage\adapter;
+
+use phpbb\storage\exception\exception;
+use phpbb\filesystem\exception\filesystem_exception;
+
+class local implements adapter_interface
```

Comment:
The storage class is not an adapter, it has the same methods, but is not exactly the same